libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/SetTestOptionsMiddleware.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Schema;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive
+{
+    /// <summary>
+    /// Middleware that catch "SetTestOptions" event and save into "Conversation.TestOptions".
+    /// </summary>
+    public class SetTestOptionsMiddleware : IMiddleware
```

Comment:
>SetTestOptionsMiddleware [](start = 17, length = 24)

This does not belong in Microsoft.Bot.Builder.Dialogs.Adaptive project.  It should be in Microsoft.Bot.Builder.Dialogs.Adaptive.Testing.